### PR TITLE
Contributor restriction and Send mail to contributor

### PR DESCRIPTION
**Pull request by ashok-ladumor**
1. Contributor roles cannot modify other’s content.
2. Contributor roles cannot publish their sell media item. (only admin can approve entries)
3. When a sale is made, send an email to the contributor who uploaded the item with their sales profit.

### CODE REVIEW

File: inc/gateways/paypal.php [hunk 1]
```diff
@@ -195,3 +195,60 @@ function sell_media_process_paypal_ipn() {
     }
 }
 add_action( 'sell_media_verify_paypal_ipn', 'sell_media_process_paypal_ipn' );
+
+/**
+* Collect comission from from cart and devide with contributors
+*/
+add_action('sell_media_after_successful_payment', 'get_commissons_by_payment_id');
+function get_commissons_by_payment_id($payment_id) {
```
Comment (thadallender) [inline inc/gateways/paypal.php:203]: Please namespace functions with sell_media_*

File: inc/gateways/paypal.php [hunk 2]
```diff
@@ -195,3 +195,60 @@ function sell_media_process_paypal_ipn() {
...
+            $_commissions_meta = get_post_meta( $_product_id, '_sell_media_commissions_meta', true );
+
+            // get contributor who assigned in product
+            $_holder = $_commissions_meta['holder'];
+
+            // Comission %
+            $_commissions = $_commissions_meta['percentage'];
+
+            // calculate row comission
+            $_finl_comission = $_row_total * $_commissions / 100;
```
Comment (thadallender) [inline inc/gateways/paypal.php:230]: Should be $_final_comission?

File: inc/gateways/paypal.php (hunk 1 above)
Comment (ashok-ladumor) [reply inc/gateways/paypal.php]: @thadallender  Added namespace.

File: inc/gateways/paypal.php (hunk 2 above)
Comment (ashok-ladumor) [reply inc/gateways/paypal.php]: @thadallender Changed

**Pull request merged**